pdns/dnsbackend.hh

```diff
@@ -86,6 +87,16 @@ struct DomainInfo
       return DomainInfo::Native;
   }
 
+  const bool isMaster(const ComboAddress& ip)
+  {
+    for( const auto& master: masters) {
+      const ComboAddress caMaster(master);
```

Comment:
this needs to compare only IP addresses, and not also the port, I'm almost sure :-)

Reply:
I almost sure you are right  ;) Fixing...